### DISCUSSION TITLE
Issue with Creating PDFs with 7.0.11-5

### DISCUSSION THREAD

**Discussion by RyanAldridge** [ImageMagick/ImageMagick#3478] [Help]
We were using 7.0.10-33 on some Windows Server 2012R2 machines to perform some simple conversions from bmp and png to PDFs. I have attempted to move to the latest version 7.0.11-5 but the conversion process is now failing. No error message is shown and a .pdf file is created however it is smaller than the pdf created by 7.0.10-33 and when attempting to open the pdf adobe reports 'There was an error opening this document. The file is damaged and could not be repaired.'. the attached image shows the source image 'Transco.bmp' and the output from the latest version 'Transco.pdf' and then after I have reverted to the old version by changing my path and re-running the command to get the 'Transco2.pdf' which is larger and opens in Adobe reader.

![image](https://user-images.githubusercontent.com/24721755/113069754-f0821900-91b8-11eb-9f9e-5e8015b6dc96.png)

**Comment by fmw42** [comment]
What was your  exact command line?  

`convert test.png test.pdf`

works fine for me and I can open it in Adobe Acrobat reader (v20.013) on my Mac OSX Sierra using ImageMagick 6.9.12.4 Q16

ImageMagick 6.9.12-4 Q16 x86_64 2021-03-20 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC Modules OpenMP(3.1) 
Delegates (built-in): bzlib cairo djvu fftw fontconfig freetype gslib gvc jbig jng jp2 jpeg lcms lqr ltdl lzma openexr png ps raw rsvg tiff webp x xml zlib

**Reply by RyanAldridge** [reply]
The exact command is shown in the screenshot. Thought admittedly the screenshot shows the conversion working after I went back to 7.0.10, the command was the same one that created a corrupt pdf with 7.0.11 : Magick transco.bmp transco.pdf

**Comment by dlemstra** [comment]
Please share your input image so we can try to reproduce this much better. And can you also double check if you can reproduce it with version `7.0.11-6`?
